changelog.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - [Change default username for client initialization to `root`](https://github.com/ballerina-platform/ballerina-standard-library/issues/2397)
 - [Fix incorrect retrieval of time types](https://github.com/ballerina-platform/ballerina-standard-library/issues/3023)
+- [Fix NullPointerException when retrieving record with default value](https://github.com/ballerina-platform/ballerina-standard-library/issues/2985)
```

Comment:
Lets update the heading, since these are now released.

Reply:
Fixed in [5f323b2](https://github.com/ballerina-platform/module-ballerinax-mysql/pull/536/commits/5f323b274565bfef2134f91af4ae9e21bbe3be5e)